### PR TITLE
Feature/database and requests

### PR DESCRIPTION
**Pull request by berciuliviu**
Add backend project which ElephantSQL database link, HTTP requests, CORS policy

### CODE REVIEW

File: ResourceSharing/WebApplication2/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -1,47 +1,46 @@
-﻿using System;
```
Comment (LeancaBogdan) [inline ResourceSharing/WebApplication2/Controllers/ValuesController.cs]: Please delete this file, as it is no longer needed

File: ResourceSharing/WebApplication2/Migrations/20181115234742_AddAllModels.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (LeancaBogdan) [inline ResourceSharing/WebApplication2/Migrations/20181115234742_AddAllModels.cs]: Is this migration needed?

File: ResourceSharing/WebApplication2/Migrations/ResourcesContextModelSnapshot.cs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+using WebApplication2;
+
+namespace WebApplication2.Migrations
+{
+    [DbContext(typeof(ResourcesContext))]
+    partial class ResourcesContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```
Comment (LeancaBogdan) [inline ResourceSharing/WebApplication2/Migrations/ResourcesContextModelSnapshot.cs]: Please add a comment on this line specifying the reason why these warnings are disabled

Comment (berciuliviu) [reply ResourceSharing/WebApplication2/Migrations/ResourcesContextModelSnapshot.cs]: It was generated automatically, so i have no idea, not under my jurisdiction

File: ResourceSharing/WebApplication2/Controllers/ValuesController.cs (hunk 1 above)
Comment (berciuliviu) [reply ResourceSharing/WebApplication2/Controllers/ValuesController.cs]: will do

Comment (berciuliviu) [reply ResourceSharing/WebApplication2/Controllers/ValuesController.cs]: I did not find the file in the current branch, it is from another

**Pull request merged**